README.md

```diff
@@ -3,29 +3,48 @@
 [![NPM Package](https://img.shields.io/npm/v/@openzeppelin/test-helpers.svg)](https://www.npmjs.org/package/@openzeppelin/test-helpers)
 [![Build Status](https://travis-ci.com/OpenZeppelin/openzeppelin-test-helpers.svg?branch=master)](https://travis-ci.com/OpenZeppelin/openzeppelin-test-helpers)
 
-**JavaScript testing helpers for Ethereum smart contract development.** With support for Truffle and plain web3.js workflows.
+*Assertion library for Ethereum smart contract testing.* Make sure your contracts behave as expected!
```

Comment:
We should use bold instead of italics, it's much more visible.

I don't like the "make sure" wording, it sounds like it's a prerequisite to using the library.  "Ensure" sounds better, I think, but I would definitely remove the exclamation mark.

Reply:
I tried to use bold, it seems single stars make text bold on asciidoc, but not on markdown.